_rules/SC1-2-1-video-only-alternative.md

```diff
@@ -0,0 +1,190 @@
+
+### Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+
+- [media alternative video](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-1-media-alternative-video.html)
+- [video only description track](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-1-video-only-description-track.html)
+- [video only element transcript](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-1-video-only-element-transcript.html)
+- [video only audio alternative](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-1-video-only-audio-alternative.html)
+
+## Assumptions
+
+This rule assumes that a mechanism is available to start the video and that the video element is not simply used to display the poster.
+
+## Accessibility support
+
+See [Video only element description track: accessibility support](https://auto-wcag.github.io/auto-wcag/rules/SC1-2-1-Video-only-description-track.html#accessibility-support).
+
+## Background
+
+- [Understanding SC 1.2.1:Audio-only and Video-only (Prerecorded)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/media-equiv-av-only-alt.html)
+- [H96:Using the track element to provide audio descriptions](https://www.w3.org/TR/WCAG20-TECHS/H96.html) 
+
+## Test Cases
+
+### Passed
+
+#### Pass example 1
+
+A video element without audio. The text on the page labels the video as an alternative.
+
+```html
+<p>Not being able to use your computer because your mouse 
+  doesn't work, is frustrating. Many people use only the keyboard to navigate 
+  websites. Either through preference or circumstance. This is solved by keyboard compatibility. 
+  Keyboard compatibility is described in WCAG.
+  See the video below to watch the same information again in video form.</p>
+<video data-rule-target src="../test-assets/perspective-video/perspective-video-with-captions-silent.mp4" controls>
+</video>
+```
+
+#### Pass example 2
+
+A video only element with a track element that contains descriptions.
+
+```html
+<video controls>
+  <source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4" />
+  <source src="../test-assets/rabbit-video/silent.webm" type="video/webm" />
+  <track kind="descriptions" src="rabbit-video-descriptions.vtt" />
+</video>
+```
+
+#### Pass example 3
+
+A silent video element with a text transcript on the same page.
+
+```html
+<video controls data-rule-target>
+  <source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4"></source>
+  <source src="../test-assets/rabbit-video/silent.webm" type="video/webm"></source>
+</video>
+<p>The above video shows a giant fat rabbit climbing out of a hole in the ground. 
+He stretches, yaws, and then starts walking. 
+Then he stops to scratch his bottom.</p>
+```
+
+#### Pass example 4
+
+A video element without audio has a seperate audio track that describes the visual information.
```

Comment:
replace seperate with separate (in the other rules also)

Reply:
Done. Thanks!